src/pages/Dashboard.jsx

```diff
@@ -17,72 +19,91 @@ const LAST_THREE_GROUPS = MOCK_GROUP_INFO.slice(-3);
 
 
 const MOCK_EXPENSES = [
-    {expenseName: "Plane tickets to Hawaii", group: "Hawaii Trip", amount: 150, id: "expense_id_1"},
-    {expenseName: "Hotel for Bali", group: "Bali Trip", amount: 150, id: "expense_id_2"},
-    {expenseName: "Beer for John's Birthday", group: "John's Birthday", amount: 150, id: "expense_id_3"},
-    {expenseName: "BBQ for Mike's Birthday", group: "Mike's Birthday", amount: 150, id: "expense_id_4"},
-    {expenseName: "Escape Room Entry fee", group: "Escape Room", amount: 150, id: "expense_id_5"},
+    { name: "Plane tickets to Hawaii", group: "Hawaii Trip", amount: 150, id: "expense_id_1", date: new Date('2024-01-02') },
+    { name: "Hotel for Bali", group: "Bali Trip", amount: 150, id: "expense_id_2", date: new Date('2024-08-23') },
+    { name: "Beer for John's Birthday", group: "John's Birthday", amount: 150, id: "expense_id_3", date: new Date('2024-05-18') },
+    { name: "BBQ for Mike's Birthday", group: "Mike's Birthday", amount: 150, id: "expense_id_4", date: new Date('2024-06-30') },
+    { name: "Escape Room Entry fee", group: "Escape Room", amount: 150, id: "expense_id_5", date: new Date('2024-02-24') },
 ]
 
+//Sorting expenses by date
+const SORTED_MOCK_EXPENSES = MOCK_EXPENSES.sort((a, b) => b.date - a.date);
+
+//Last 3 expenses
+const LAST_THREE_EXPENSES = SORTED_MOCK_EXPENSES.slice(-3);
+
+
 
 
 function Dashboard() {
     return (
         <div className="w-full px-4">
-            <h1 className="text-2xl md:text-5xl">Dashboard</h1>
-            <section className=" my-4 flex w-full justify-center flex-wrap">
+            <Heading tag="h1" className="">Dashboard</Heading>
+            <section className=" my-4 flex w-full justify-center flex-wrap md:flex-nowrap md:gap-2">
                 <Card className="w-full">
-                    <CardHeader className="flex flex-row">
-                        <Group />
-                        <h1 className="flex flex-col">
-                            <span>Total Groups</span>
-                            <span>{MOCK_GROUP_INFO.length}</span>
-                        </h1>
+                    <CardHeader className="flex flex-row my-auto">
+                        <Group size={64} className="pr-3 border-r-2" />
+                        <div className="flex flex-col ml-4">
```

Comment:
please use the `BodyText` component instead of `Heading` for better accessibility and SEO. Consider using font-bold for titles and normal font weight for descriptions.